### ISSUE TITLE
Better tagline / updates to website

### ISSUE THREAD

**Issue by mgsloan** [xmonad/xmonad#166]
Not sure if this is the right place for nitpicks, but I think a couple little tweaks would help make xmonad look a bit better to people:

The github tagline mentions "ICCCM compliant".  I doubt most people know what this means or care, I certainly didn't.  How about just saying "X11"?  That is good info so people won't expect it to work with wayland, if they know that distinction.

Also, I took a look at xmonad.org and having testimonials from nearly 10 years ago doesn't make xmonad seem up to date / maintained.

So yeah, just minor nitpicks.  I can make the changes if given access :D

**Comment by liskin** [comment]
Are you still interested in helping out? https://github.com/xmonad/xmonad-web is where we keep the website sources, so just send a PR.

Or, if you prefer, we can just give you push access I guess.

**Comment by slotThe** [comment]
I think the "A dynamically tiling X11 window manager that is written and configured in [Haskell](https://www.haskell.org/)." thing that we have on the website is pretty good already, so from my side only the GitHub description remains that may be changed.  However, "X11" probably means "EWMH compliant" in a lot of people's minds (not in those terms, but they certainly expect EWMH compliancy, I would imagine!)

**Comment by geekosaur** [comment]
Yes, the ICCCM stress is to distinguish from EWMH… but it won't be widely understood, sadly. That said, perhaps in 2022 it's time to let go of sjanssen's belief that xmonad was feature complete when it could display 20 xterms. :)

**Comment by slotThe** [comment]
I think with https://github.com/xmonad/xmonad-contrib/pull/625 landing it would be quite hard to move EWMH into the core, so perhaps that's a good excuse for this to stay there.

On the other hand, the tutorial shortly [mentions this](https://github.com/xmonad/xmonad/blob/master/TUTORIAL.md?plain=1#L358-L362) and then suggests that users use `emwh` to save themselves some headache further down the line.  Perhaps a GitHub project description doesn't have to be 100% accurate if that means more people can understand what's going on

**Comment by geekosaur** [comment]
That's sad. Maybe it's a reason to keep `XMonad.Config.Desktop` around, though: add `docks` and maybe `ewmhFullscreen` to it and suggest it as the quick way to get a modern desktop going.
